### PR TITLE
ceph-volume: check if we run in an selinux environment, now also in py2

### PR DESCRIPTION
**Pull request by jan--f**
Fixes: https://tracker.ceph.com/issues/42967

### CODE REVIEW

**Comment by jan--f** [comment]
jenkins test ceph-volume tox

File: src/ceph-volume/ceph_volume/tests/util/test_system.py [hunk 1]
```diff
@@ -226,6 +226,13 @@ def apply(value='/bin/restorecon'):
     return apply
 
 
+# python2 has no FileNotFoundError
```
Comment (alfredodeza) [inline src/ceph-volume/ceph_volume/tests/util/test_system.py]: I take it that this should've failed with tox, just that we didn't run it before right?

Comment (jan--f) [reply src/ceph-volume/ceph_volume/tests/util/test_system.py]: correct

**Comment by jan--f** [comment]
tox failure is addressed in #31816

**Pull request merged**